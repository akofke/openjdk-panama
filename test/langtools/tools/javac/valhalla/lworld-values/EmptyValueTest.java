/*
 * @test /nodynamiccopyright/
 * @summary Test behavior with empty value type.
 * @compile/fail/ref=EmptyValueTest.out -XDrawDiagnostics -XDdev EmptyValueTest.java
 * @compile -XDrawDiagnostics -XDdev -XDallowEmptyValues EmptyValueTest.java
 */
public final value class EmptyValueTest {
    static int x;
}
