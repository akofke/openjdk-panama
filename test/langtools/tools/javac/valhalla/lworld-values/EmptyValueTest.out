EmptyValueTest.java:7:24: compiler.err.empty.value.not.yet
1 error